styles.css

```diff
@@ -261,21 +263,21 @@ hr.works-line {
   padding-left: 25px;
   padding-right: 15px;
   top: 1px;
-  font-family: Inter;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: normal;
   font-size: 18px;
   line-height: 24px;
   flex: none;
   order: 1;
   flex-grow: 0;
-  margin: 12px 0px;
+  margin: 12px 0.01px;
   margin: auto;
 }
 
 .stories-lang-list {
   /* Auto Layout */
-  
+
   display: grid;
```

Comment:
- [ ] kindly use the font,  margins (top, bottom, left, right) defined in the design. It is best practice to pay attention to the details given.
- your design
![Screenshot from 2021-09-25 07-34-26](https://user-images.githubusercontent.com/65860678/134758482-588ab7d2-21df-472f-ad28-3155cb3d4be7.png)
-Figma design
![Screenshot from 2021-09-25 07-47-01](https://user-images.githubusercontent.com/65860678/134758509-d5ddd879-2e5a-4db4-a978-41ff43e25766.png)